WebApplication/ClientApp/src/components/modalFail.test.js

```diff
@@ -119,10 +119,6 @@ describe('modal update failed ', () => {
             // validate title
             const titleBlock = wrapper.find('.errorMessageTitle');
             expect(titleBlock.exists()).toEqual(!! errorData.title);
-            if (errorData.title) {
```

Comment:
Can you explain this change?

Reply:
Yes, this was a build error saying that **conditional expect**s are now prohibited.
[https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md#:~:text=Jest%20considered%20a%20test%20to%20have%20failed%20if,needed%20to%20understand%20what%20is%20actually%20being%20tested.](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md#:~:text=Jest%20considered%20a%20test%20to%20have%20failed%20if,needed%20to%20understand%20what%20is%20actually%20being%20tested.)

Reply:
IMHO - it's better to suppress ESLint warning here, not to remove the test

Reply:
Exactly as @erymski said. We can now suppress error through the lint ignore. And for us it can be tech. dept we can resolve later.

Also it would be nice to understand why this is conditional. To answer question how it is possible errorData.title can be falsy.

Reply:
I couldn't find the way how to disable this BABEL warning it doesn't seem to be configurable. I rewrote it with the recommended approach similar to examples on their site.